tests/test_sim_stimulator.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020 Simons Observatory.
+# Full license can be found in the top level "LICENSE" file.
+
+"""Check functionality of the wiregrid simulation
+
+"""
+
+import unittest
+import numpy as np
+
+import astropy.units as u
+import toast
+
+import sotodlib.toast as sotoast
+import sotodlib.toast.ops as so_ops
+
+
+class SimStimulatorTest(unittest.TestCase):
+    def test_instantiate(self):
+        """Test instantiating a simulation operator."""
+        comm, procs, rank = toast.get_world()
```

Comment:
For all of the unit test classes using toast, we should guard against import errors in case toast is not installed.  This is the technique I used in the toast / data PR (note the debugging `raise` statement in there which should be replaced by `toast_available = False`):  https://github.com/simonsobs/sotodlib/blob/4e4fb294884d5e7d859260631b5676da0e866832/tests/test_toast_books.py#L24-L46

Reply:
Added import guard.